app/views/step_by_step_pages/show.html.erb

```diff
@@ -21,12 +21,17 @@
     <% unless @step_by_step_page.can_be_published? %>
       <%= render "govuk_publishing_components/components/inset_text" do %>
         <h2 class="govuk-heading-m">To publish this step by step you need to</h2>
+
         <% if !@step_by_step_page.steps.any? %>
           <%= link_to('Add at least one step', new_step_by_step_page_step_path(@step_by_step_page), class: 'govuk-link govuk-link--no-visited-state') %>
-        <% elsif !@step_by_step_page.steps_have_content? %>
-          <%= link_to('Add content to all your steps', '#steps', class: 'govuk-link govuk-link--no-visited-state') %>
-        <% elsif !@step_by_step_page.links_checked_since_last_update? %>
-          <%= link_to "Check for broken links", { action: "check_links", step_by_step_page_id: @step_by_step_page.id}, method: :post, class: 'govuk-link govuk-link--no-visited-state' %>
+        <% else %>
+          <% unless @step_by_step_page.steps_have_content? %>
+            <%= link_to('Add content to all your steps', '#steps', class: 'govuk-link govuk-link--no-visited-state') %>
+          <% end %>
+
+          <% unless @step_by_step_page.links_checked_since_last_update? %>
```

Comment:
Same here. Using a nested `unless` is hard to read. `links_checked_since_last_update?` would be better being renamed something like `check_links?` as I suggested yesterday, and then it could be called with an `if` and moved out of the nested conditionals.